jenny/templates/pvc.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if .Values.persistence.enabled }}
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ .Values.volumes.media.claimName }}
+  namespace: default
+spec:
+  accessModes:
+  {{ with .Values.persistence.accessModes }}
+    {{- toYaml . | indent 4 }}
+  {{- end }}
+  storageClassName: {{ .Values.persistence.storageClass }}
+  resources:
+    requests:
+      storage: 2Gi
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ .Values.volumes.static.claimName }}
+  namespace: default
```

Comment:
The namespace should be configurable

Reply:
resolved by https://github.com/openfun/charts/pull/4/commits/0821c6865a9571ce15158490236b4babf84f84f1